socketioxide/src/lib.rs

```diff
@@ -61,7 +61,8 @@
 //! ```
 
 pub mod adapter;
-pub mod retryer;
+pub mod errors;
```

Comment:
Only export needed error like below and not the whole mod

Reply:
I didnt get you. I used this mod in example to handle errors. Did you mean it's needed to split into pub errors and private to different modules?

Reply:
Yes, just like line 68 where it is already done (only `AckError` and `Error` are exported from root level)